.golangci.yml

```diff
@@ -0,0 +1,4 @@
+issues:
+  exclude:
```

Comment:
@chenrui333 could you add a comment explaining why we need to exclude these checks? It's not obvious to me, and I don't want the context to get lost :)

Reply:
Sure, I am doing that now.

Reply:
Done. :)

Reply:
Let me know if you need more things from my side.